src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1522,27 +1522,14 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+            <include cluster="Binding" client="false" server="true" clientLocked="true" serverLocked="true">
                 <requireAttribute>BINDING</requireAttribute>
             </include>
-            <include cluster="Groups" client="false" server="true" clientLocked="true" serverLocked="true">
-                <requireAttribute>GROUP_NAME_SUPPORT</requireAttribute>
-                <requireCommand>AddGroup</requireCommand>
-                <requireCommand>AddGroupResponse</requireCommand>
-                <requireCommand>ViewGroup</requireCommand>
-                <requireCommand>ViewGroupResponse</requireCommand>
-                <requireCommand>GetGroupMembership</requireCommand>
-                <requireCommand>GetGroupMembershipResponse</requireCommand>
-                <requireCommand>RemoveGroup</requireCommand>
-                <requireCommand>RemoveGroupResponse</requireCommand>
-                <requireCommand>RemoveAllGroups</requireCommand>
-                <requireCommand>AddGroupIfIdentifying</requireCommand>
-            </include>
-            <include cluster="Scenes" client="true" server="false" clientLocked="true" serverLocked="true">
-            </include>
+            <include cluster="Groups" client="false" server="false" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Scenes" client="false" server="false" clientLocked="false" serverLocked="true"></include>
             <include cluster="On/Off" client="true" server="false" clientLocked="true" serverLocked="true"></include>
-            <include cluster="Level Control" client="true" server="false" clientLocked="false" serverLocked="true"></include>
-            <include cluster="Color Control" client="true" server="false" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Level Control" client="false" server="false" clientLocked="false" serverLocked="true"></include>
+            <include cluster="Color Control" client="false" server="false" clientLocked="false" serverLocked="true"></include>
```

Comment:
I see server attributes mentioned under some cluster and not present under other clusters. Is this required in the xml here?

Reply:
If attributes mentioned cluster adds all mentioned attributes and if not it adds only mandatory attributs.